docs/data/migrate-from-horizon-to-rpc/README.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Migrate from Horizon to RPC
+sidebar_position: 0
+---
+
+Applications using [Horizon's REST-like API] will need to be updated to use the [RPC JSON-RPC API] when migrating from [Horizon] to [RPC]. This guide provides an overview of the key differences between the two APIs and how to migrate your application.
```

Comment:
I'm not going to separately comment on all occurrences, but looks like there's a bunch of places you intended to put links so left `[]`, but forgot to put the actual addresses in.

Reply:
Ah nevermind, I just saw the references below. Problem is that those aren't rendering properly in the preview

Reply:
Appears to be rendering fine after merging main into the branch.